sdk/src/runtime/requestInfo/utils.tsx

```diff
@@ -0,0 +1,54 @@
+import { FC } from "react";
+import { type DocumentProps } from "../lib/types";
+import { generateNonce } from "../lib/utils";
+import { type PartialRequestInfo, type RequestInfo } from "./types";
+
+export const DefaultRequestInfoDocument: FC<DocumentProps> = ({ children }) => (
+  <>{children}</>
+);
+
+/**
+ * Constructs a generic requestInfo that can be used as defaults.
+ * Allows for passing in overrides to initialize with defaults.
+ */
+export const constructWithDefaultRequestInfo = (
+  overrides: PartialRequestInfo = {},
+): RequestInfo => {
+  const { rw: rwOverrides, ...otherRequestInfoOverrides } = overrides;
+
+  const defaultRequestInfo: RequestInfo = {
+    request: new Request("http://localhost/"),
+    params: {},
+    ctx: {},
+    cf: {
+      waitUntil: () => {},
+      passThroughOnException: () => {},
+      props: {},
+    },
```

Comment:
Ah, I guess we need this stubbed this way since the caller wouldn't be providing this, makes sense 👍